snippets/snippets_setup.R

```diff
@@ -1,3 +1,11 @@
+# Only install missing libraries, per https://stackoverflow.com/a/23286257/4138705.
+install_if_missing <- function(packages) {
+  if (length(setdiff(packages, rownames(installed.packages()))) > 0) {
+    install.packages(setdiff(packages, rownames(installed.packages())))
+  }
+}
+install_if_missing(c('virids', 'ggthemes', 'scales', 'skimr', 'lubridate', 'bigrquery', 'tidyverse'))
```

Comment:
virids -> viridis

Reply:
Oops - nice catch. I fixed this when I was testing this out locally, then completely forgot to backport the fix into my PR.